### PR TITLE
DB-8714 Factor out common SpliceDefaultCompactor functions

### PR DESCRIPTION
**Pull request by arnaud-splice**

### CODE REVIEW

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by arnaud-splice** [comment]
jenkins please retest branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by arnaud-splice** [comment]
jenkins please retest branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by arnaud-splice** [comment]
jenkins please retest branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Comment by arnaud-splice** [comment]
jenkins please test branch @mapr6.1.0

**Comment by spliceci** [comment]
TEST FAILED

**Comment by arnaud-splice** [comment]
jenkins please retest branch @mapr6.1.0

**Comment by spliceci** [comment]
TEST FAILED

**Comment by arnaud-splice** [comment]
jenkins please test branch @hdp2.6.1

**Comment by spliceci** [comment]
TEST FAILED

File: hbase_sql/hbase1.1.1/src/main/java/com/splicemachine/compactions/SpliceDefaultCompactor.java [hunk 1]
```diff
@@ -16,83 +16,45 @@
...
-
-        if (LOG.isDebugEnabled()) {
-            SpliceLogUtils.debug(LOG, "Initializing compactor: region=%s", ((HStore)this.store).getHRegion());
-        }
     }
 
     public SpliceDefaultCompactor(final Configuration conf, final Store store, long smallestReadPoint) {
-        this(conf, store);
-        this.smallestReadPoint = smallestReadPoint;
+        super(conf, store);
```
Comment (yxia92) [inline hbase_sql/hbase1.1.1/src/main/java/com/splicemachine/compactions/SpliceDefaultCompactor.java]: Shouldn't we call super(conf, store, smallestReadPoint) instead?

Comment (arnaud-splice) [reply hbase_sql/hbase1.1.1/src/main/java/com/splicemachine/compactions/SpliceDefaultCompactor.java]: Good point, thank you for catching that one!

**Pull request merged**